web-ng/src/ground-maps/ground-maps.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'ground-maps',
+  templateUrl: './ground-maps.ng.html',
+  styleUrls: ['./ground-maps.css'],
+})
+export class GroundMaps {
```

Comment:
It seems the style guide recommends a prefix for area of the app, but not for the app name itself:

> Do use a prefix that identifies the feature area or the app itself.
> 
> Why? Prevents element name collisions with components in other apps and with native HTML elements.
> 
> Why? Makes it easier to promote and share the component in other apps.
> 
> Why? Components are easy to identify in the DOM.

Source: https://angular.io/guide/styleguide#component-custom-prefix

So in this we'd probably have a bunch of "Map" components, e.g. `MapContainer`, `MapControls`, etc.